### PR TITLE
update with multi-node mode

### PR DESCRIPTION
**Pull request by deep-diver**
This PR extends the existing example on Alignment Handbook on multiple node configurations.

### CODE REVIEW

**Comment by peterschmidt85** [comment]
@deep-diver I don't see the multi-node task configuration

File: examples/fine-tuning/alignment-handbook/README.md [hunk 1]
```diff
@@ -105,7 +105,65 @@ WANDB_API_KEY=<...> \
...
+      --main_process_ip=$DSTACK_MASTER_NODE_IP \
+      --main_process_port=8008 \
+      --machine_rank=$DSTACK_NODE_RANK \
+      --num_processes=$DSTACK_GPUS_NUM \
+      --num_machines=$DSTACK_NODES_NUM \
+      scripts/run_sft.py recipes/custom/config.yaml
+ports:
+  - 6006 
+resources:
+  gpu: 1..2
```
Comment (peterschmidt85) [inline examples/fine-tuning/alignment-handbook/README.md]: Just curious, why `1..2`? Shall it be `24GB:2`?

Comment (deep-diver) [reply examples/fine-tuning/alignment-handbook/README.md]: Sure. No problem! 
If there is a way to configure this at runtime, that would be much better IMO

**Comment by deep-diver** [comment]
> @deep-diver I don't see the multi-node task configuration

What do you mean?

**Comment by peterschmidt85** [comment]
> What do you mean?
@deep-diver I mean you have only `train.dstack.yaml` with a single node but you don't have a `train-distrib.dstack.yaml` with multiple nodes as well as other configuration files.
As a user how do I run the multi-node configuration with a single command without writing the configuration file myself?

File: examples/fine-tuning/alignment-handbook/README.md (hunk 1 above)
Comment (peterschmidt85) [reply examples/fine-tuning/alignment-handbook/README.md]: My preference is always the declarative approach over runtime. In the runtime, the user always can override if needed.

**Comment by deep-diver** [comment]
@peterschmidt85 updated!

**Pull request merged**